www/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
```

Comment:
Your change should not include any www-related changes, please.  Please revert this file in your next commit.